project_scrum/models/project_scrum_us.py

```diff
@@ -0,0 +1,144 @@
+        inverse_name='us_id',
+    )
+    task_test_ids = fields.One2many(
+        comodel_name='project.scrum.test',
+        inverse_name='user_story_id_test',
+    )
+    task_count = fields.Integer(compute='_compute_task_count', store=True)
+    test_ids = fields.One2many(
+        comodel_name='project.scrum.test',
+        inverse_name='user_story_id_test',
+    )
+    test_count = fields.Integer(compute='_compute_test_count', store=True)
+    sequence = fields.Integer()
+    company_id = fields.Many2one(
+        related='project_id.analytic_account_id.company_id',
+        store=True
+    )
+    moscow = fields.Selection('_get_moscow_field', string='Moscow')
+    value = fields.Selection('_get_value_field', string='Value')
+    risk = fields.Selection('_get_risk_field', string='Risk')
+    kano = fields.Selection('_get_kano_field', string='Kano')
+    reference = fields.Char(
+        'Number',
+        index=True,
+        readonly=True,
+        copy=False,
+        default='/',
+    )
+    kanban_state = fields.Selection([
+        ('normal', 'Mark as impeded'),
+        ('blocked', 'Mark as waiting'),
+        ('done', 'Mark item as defined and ready for implementation')],
+        'Kanban State',
+        default='blocked',
+    )
+
+    @api.depends("task_ids")
+    def _compute_task_count(self):
+        for p in self:
+            p.task_count = len(p.task_ids)
+
+    @api.depends("test_ids")
+    def _compute_test_count(self):
+        for p in self:
+            p.test_count = len(p.test_ids)
+
+    def _resolve_project_id_from_context(self):
+        """ Returns ID of project based on the value of 'default_project_id'
+            context key, or None if it cannot be resolved to a single
+            project.
+        """
+        context = self.env.context
+        project_project_model = self.env['project.project']
+        if type(context.get('default_project_id')) in (int, long):
+            return context['default_project_id']
+        if isinstance(context.get('default_project_id'), basestring):
+            project_name = context['default_project_id']
+            project_ids = project_project_model.with_context(context)
+            project_ids = project_ids.name_search(project_name, operator='=')
+            if not project_ids:
+                project_ids = project_ids.name_search(
+                    project_name,
+                    operator='=ilike',
+                )
+            if not project_ids:
+                project_ids.name_search(name=project_name)
+            if len(project_ids) == 1:
+                return project_ids[0][0]
+        return None
+
+    @api.model
+    def _read_group_sprint_id(self, present_ids, domain, **kwargs):
+        project_id = self._resolve_project_id_from_context()
+        sprints = self.env['project.scrum.sprint'].search(
+            [('project_id', '=', project_id)], order='sequence').name_get()
+        return sprints, None
+
+    _group_by_full = {
```

Comment:
this is a relict from 8.0: https://github.com/OCA/OCB/blob/8.0/openerp/models.py#L1848

You probably need https://github.com/OCA/OCB/blob/10.0/odoo/models.py#L1671 here